frigate/stats.py

```diff
@@ -13,11 +15,24 @@
 logger = logging.getLogger(__name__)
 
 
+def get_latest_version() -> str:
+    try:
+        response = requests.get('https://api.github.com/repos/blakeblackshear/frigate/releases/latest').json()
+
+        if response:
+            return response.get("tag_name", "unknown").replace("v", "")
+        else:
+            return "unknown"
+    except URLError:
```

Comment:
The requests library doesn't throw this exception to my knowledge. Non-200 responses don't throw exceptions. I think you want to use 
```python
if response.ok:
```